static/src/javascripts/bootstraps/football.js

```diff
@@ -226,7 +226,10 @@ define([
         });
 
         page.isCompetition(function (competition) {
-            renderTable(competition, extras, dropdownTemplate);
+            var $rightHandCol = $('.js-secondary-column')
+            if ($rightHandCol.dim().height > 1800 ) {
```

Comment:
Maybe leave a comment above this conditional to explain why we are having to do this.